freqtrade/strategy/strategy_helper.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+from freqtrade.exchange import timeframe_to_minutes
+
+
+def merge_informative_pair(dataframe: pd.DataFrame, informative: pd.DataFrame,
+                           timeframe: str, timeframe_inf: str, ffill: bool = True) -> pd.DataFrame:
+    """
+    Correctly merge informative samples to the original dataframe, avoiding lookahead bias.
+
+    Since dates are candle open dates, merging a 15m candle that starts at 15:00, and a
+    1h candle that starts at 15:00 will result in all candles to know the close at 16:00
+    which they should not know.
+
+    Moves the date of the informative pair by 1 time interval forward.
+    This way, the 14:00 1h candle is merged to 15:00 15m candle, since the 14:00 1h candle is the
+    last candle that's closed at 15:00, 15:15, 15:30 or 15:45.
+
+    Assuming inf_tf = '1d' - then the resulting columns will be:
+    date_1d, open_1d, high_1d, low_1d, close_1d, rsi_1d
+
+    :param dataframe: Original dataframe
+    :param informative: Informative pair, most likely loaded via dp.get_pair_dataframe
+    :param timeframe: Timeframe of the original pair sample.
+    :param timeframe_inf: Timeframe of the informative pair sample.
+    :param ffill: Forwardfill missing values - optional but usually required
+    """
+
+    minutes_inf = timeframe_to_minutes(timeframe_inf)
+    if timeframe == timeframe_inf:
```

Comment:
Should this comparison be `<=` ? 

I guess shifting the candle does not make sense if the informative timeframe is smaller (e.g. merging 5m candles on top of 15m candles


i should be able to use the candle from 10:00 for both 5m and 15m - as the 5m candle stops at 10:05, while the 15m candle stops at 10:15 - so no lookahead bias (if the 15m candle is "My" timeframe).